### PR TITLE
updated for flow rates and tou controls

### PR DESCRIPTION
**Pull request by atimgraves**
allow for setting flow rates for manual mode and to provide extra functionality for the soon to be updated ha services integration

### CODE REVIEW

**Comment by atimgraves** [comment]
Jan, this is pretty much unchanged except for the flow rates code. Once you've run it and tested it to ensure it works properly and I hope released it I'll do some final testing with it against my updated version of your sonnen HA repo 
That provides support for : 
operating mode (retrieval and a service for setting)
battery reserve level (retrieval and various services for absolute and relative to the current battery level setting)
Time of use schedule (for retrieval and setting - currently only one time of use window is supported in the code)
control of flow rate (set charge and discharge rates when in manual mode)

File: sonnenbatterie/sonnenbatterie.py [hunk 1]
```diff
@@ -4,8 +4,9 @@
 # pylint: disable=unused-wildcard-import
 from .const import *
 # pylint: enable=unused-wildcard-import
+from .timeofuse import timeofuseschedule
 
-
+print("Loading sonnenBatterie")
```
Comment (weltmeyer) [inline sonnenbatterie/sonnenbatterie.py]: i think this print statement is executed everytime the module is loaded. Is this good practice in python?
(i am coming from dotnet...)

Comment (atimgraves) [reply sonnenbatterie/sonnenbatterie.py]: I have no idea, it was a debug thing for me, I'll remove it and recommit
Done

**Comment by weltmeyer** [comment]
testet get_methods on my battery sucessfully. merging and releasing :)

**Comment by weltmeyer** [comment]
https://pypi.org/project/sonnenbatterie/0.2.3/

**Pull request merged**